analyses/methylation-summary/tools/03-methyl-tpm-correlation.cwl

```diff
@@ -0,0 +1,59 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: 03-methyl-tpm-correlation
+doc: >-
+  Calculate representative probe-level correlations between rna-seq (tpm) and methylation (beta/m-vlaues) for patients who have samples in both datasets
+
+requirements:
+  - class: ShellCommandRequirement
+  - class: DockerRequirement
+    dockerPull: "alexsickler/open-pedcan:latest"
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    ramMin: ${return inputs.ram * 1000}
+    coresMin: 4
+  - class: LoadListingRequirement
+    loadListing: shallow_listing
+  - class: InitialWorkDirRequirement
+    listing:
+      - entryname: 03-methyl-tpm-correlation.py
+        entry:
+          $include: ../03-methyl-tpm-correlation.py
+
+baseCommand: [mkdir]
+
+arguments:
+  - position: 1
+    shellQuote: false
+    valueFrom: |-
+      ./.git
+      mkdir -p ./.git/objects
```

Comment:
These are created because the scripts in open ped can try to find the data and module dir based on searching the directory tree until it finds the `.git` directory which is created at the root dir of a git repo. In R, the command just looks for a directory containing a directory called `.git` however, GitPython checks for a few more directories and files that are also created by git. So the tools are creating these fake git directories so GitPython can find the corret root dir.